<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>755</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>5</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>531</width>
    <height>366</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="2">
     <widget class="QPushButton" name="open_model_button">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Open Model</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="model_label">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Model: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="input_line_edit_model">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>10</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="input_line_edit_doc">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>10</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="document_label">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Document:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="open_doc_button">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Open Doc</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QTabWidget" name="tabwidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="query_tab">
       <attribute name="title">
        <string>Query</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="0" rowspan="2" colspan="2">
         <widget class="QTextEdit" name="textOutput">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="run_query_button">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Calculate post-edit distance score</string>
          </property>
          <property name="text">
           <string>Run Query</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="train_tab">
       <attribute name="title">
        <string>Train</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="2" column="0" rowspan="3" colspan="3">
         <widget class="QTextEdit" name="textOutput_train">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="save_model_button">
          <property name="text">
           <string>Save Project</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Select:</string>
          </property>
          <widget class="QPushButton" name="train_model_button">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>100</y>
             <width>202</width>
             <height>23</height>
            </rect>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Calculate post-edit distance score</string>
           </property>
           <property name="text">
            <string>Train Model</string>
           </property>
          </widget>
          <widget class="QPushButton" name="add_documents_button">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>20</y>
             <width>202</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Add Document</string>
           </property>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>60</y>
             <width>47</width>
             <height>13</height>
            </rect>
           </property>
           <property name="text">
            <string>OR</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>870</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Model"/>
    <addaction name="separator"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHowTo"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen_Model">
   <property name="text">
    <string>Open Project...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionHowTo">
   <property name="text">
    <string>HowTo</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
